recipes-kernel/linux/files/t600/patches/0006-fix-the-cold-reset-function-with-the-new-SYSTEM-CPLD.patch

```diff
@@ -0,0 +1,73 @@
+From fd995535938a2e2be6d50b1598ac4f6e3244d1af Mon Sep 17 00:00:00 2001
+From: aken_liu <aken_liu@accton.com.tw>
+Date: Mon, 6 Aug 2018 10:21:06 +0800
+Subject: [PATCH 1/3] fix the cold reset function with the new SYSTEM CPLD
```

Comment:
Is this patch part of three patch series?

Reply:
I generate the git patch files for 3 issues at the same time. (git format-patch -3)
But, these three issue are independence. 
I will send others pull-request latter.
1. cold restart: modify file, accton_t600_cpld.c
2. the order of fan number issue: modify file, accton_t600_fan.c
3. modify MDEC control method: modify file, accton_t600_fj_mdec.c